### DISCUSSION TITLE
Components as files?

### DISCUSSION THREAD

**Discussion by aolko** [realmorg/realm#61] [Q&A] [answered]
Can components be saved and loaded as separate files, like [riot does](https://riot.js.org/documentation/#quick-start)?

**Comment by oknoorap** [comment, marked as answer]
Hi @aolko thanks for trying out this framework.

The answer is yes, you can [define a custom element](https://realm.codes/references/elements/custom-element) and [import the element](https://realm.codes/references/elements/import-element)

example:
https://codesandbox.io/p/sandbox/basic-import-yrjppy?file=%2Findex.html
